day_7/10_speckled_frogs.rb

```diff
@@ -0,0 +1,54 @@
+# Print rhyme for 10 frogs AND Extension 2: Make it work for any number of frogs
+frogs = Array(1..10).reverse  # can change 10 to desired number of frogs
+
+frogs.each do |num|
+  if num == frogs[0]
+    puts "#{num} speckled frogs sat on a log"
+    puts "eating some most delicious bugs."
+    puts "One jumped in the pool where its nice and cool,"
```

Comment:
It may be easier to divide by verse, rather than include the start of the next verse within your conditional. I see how you've divided it up, so you can just use num, but it may be more readable if you use `num - 1` from time to time.